x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -35,18 +35,22 @@ export default function ({ getService }: FtrProviderContext) {
       await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
     });
 
-    it(`Should return 500 when package policy id does not exist`, async () => {
+    it(`Should return 404 when package policy id does not exist`, async () => {
```

Comment:
this was `500` but should be `404` (not found) 

we used to get policies by `getByIDs` and throw a custom error if we don't have our item. we now use `get(id)` instead, which throws an [Boom](https://github.com/hapijs/boom) error that we pass to `transformError` which makes it able to infer the correct status code/message.